test/Identity.Test/CdnScriptTaghelperTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Security.Cryptography;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Identity.Test
+{
+    public class CdnScriptTagTests
+    {
+        private readonly ITestOutputHelper _output;
+
+        public CdnScriptTagTests(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Fact]
+        public async Task IdentityUI_ScriptTags_SubresourceIntegrityCheck()
+        {
+            var slnDir = GetSolutionDir();
+            var sourceDir = Path.Combine(slnDir, "src", "UI", "Areas", "Identity", "Pages");
```

Comment:
@natemcmaster I am looking into 'src' here because IdentityUI's .cshtml files are embedded in its assembly and its a bit more involved in getting their content (even though that is the ideal thing to do in tests). We can file a follow-up issue to fix this, if needed.

Reply:
I think checking the original source is good enough for now. Would it be better to have this test glob over every 
.cshtml file under src/UI/ ?